src/universal/components/TaskEditor/TaskEditor.js

```diff
@@ -21,10 +21,7 @@ class TaskEditor extends Component {
     trackEditingComponent: PropTypes.func,
     handleBeforeInput: PropTypes.func,
     handleChange: PropTypes.func,
-    handleUpArrow: PropTypes.func,
```

Comment:
i had to PR draft to fix a read-only bug. the latest version deprecated these handlers.